File: P2Ejercicio1.py
def pys():
	print("Bienvenido, ingrese los 5 valores para obtener su sumatoria y promedio")
	V1=float(input("Ingrese el primer valor: "))
	V2=float(input("Ingrese el segundo valor: "))
	V3=float(input("Ingrese el tercer valor: "))
	V4=float(input("Ingrese el cuarto valor: "))
	V5=float(input("Ingrese el último valor: "))
	sum=V1+V2+V3+V4+V5
	prom=sum/5
	print("Sumatoria= "+str(sum))
	print("Promedio= "+str(prom))
pass

pys()

File: P2Ejercicio1.c
#include<stdio.h>

void main(void){

float a[5];
float sum=0;
float prom=0;

printf("Bienvenido, ingrese los 5 datos para obtener su promedio y la sumatoria. \n");
printf("Escriba el primer número: ");
scanf("%f",&a[0]);
printf("Escriba el segundo número: ");
scanf("%f",&a[1]);
printf("Escriba el tercero número: ");
scanf("%f",&a[2]);
printf("Escriba el cuarto número: ");
scanf("%f",&a[3]);
printf("Escriba el último número: ");
scanf("%f",&a[4]);

sum=a[0]+a[1]+a[2]+a[3]+a[4];
prom=sum/5;

printf("Sumatoria=%f. \n",sum);
printf("Promedio=%f. \n",prom);

}
